Index: src/app/map/page.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>const MapPage = () => {\n  return <></>;\n};\n\nexport default MapPage;\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/app/map/page.tsx b/src/app/map/page.tsx
--- a/src/app/map/page.tsx	(revision 753c49c2b27c72abc4b60983c5ee17625f67c496)
+++ b/src/app/map/page.tsx	(date 1747811148958)
@@ -1,5 +1,7 @@
+import CourtsPage from "@/app/court/page";
+
 const MapPage = () => {
-  return <></>;
+  return <CourtsPage />;
 };
 
 export default MapPage;
